src/bin/vip-config-software.js

```diff
@@ -2,10 +2,36 @@
 
 import command from '../lib/cli/command';
 
+const usage = 'vip config software';
+const exampleUsage = 'vip @example-app.develop config software';
+const exampleUsageNode = 'vip @example-node-app.develop config software';
+
+// Command examples
+const examples = [
+	{
+		usage: `${ exampleUsage } get`,
+		description: 'Retrieve the current software configuration for all supported components.',
+	},
+	{
+		usage: `${ exampleUsage } get wordpress --include available_versions`,
+		description:
+			'Retrieve the current software configuration for WordPress including available versions.',
```

Comment:
Suggested:
`Retrieve the current version of WordPress for a WordPress environment and a list of available versions in the default table display.`

Reply:
I'm going to use "format" in place of "display" because that's what we've used in most places and stays consistent with the argument name, `--format`

I'll make sure the examples in the other get commands are also consistent.